pyquil/paulis.py

```diff
@@ -916,8 +894,8 @@ def exponential_map(term: PauliTerm) -> Callable[[float], Program]:
 
     :param term: A pauli term to exponentiate
     :returns: A function that takes an angle parameter and returns a program.
-    :rtype: Function
     """
+    assert isinstance(term.coefficient, complex)
```

Comment:
It appears this assertion is causing the test suite to fail.

Reply:
To be removed in 2nd commit.